### PR TITLE
WRN-15494: Improve color picker integration performance.

### PR DESCRIPTION
**Pull request by alexandrumorariu**
### Checklist

* [X] I have read and understand the [contribution guide](http://enactjs.com/docs/developer-guide/contributing/)
* [ ] A [CHANGELOG entry](http://enactjs.com/docs/developer-guide/contributing/changelogs/) is included
* [ ] At least one test case is included for this feature or bug fix
* [X] Documentation was added or is not needed
* [ ] This is an API breaking change

### Issue Resolved / Feature Added
[//]: # (Describe the issue resolved or feature added by this pull request)
Color picker sliders were very laggy.

### Resolution
[//]: # (Does the code work as intended?)
[//]: # (What is the impact of this change and *why* was it made?)
Changed the color picker to apply the changes only when the Apply button is pressed. This will reduce the number of re-renders and will smoothen the app's usage.

### Additional Considerations
[//]: # (How should the change be tested?)
[//]: # (Are there any outstanding questions?)
[//]: # (Were any side-effects caused by the change?)

### Links
[//]: # (Related issues, references)
WRN-15494

### Comments

### CODE REVIEW

File: sandstone/custom-skin/src/views/MainPanel.js [hunk 1]
```diff
@@ -315,7 +315,7 @@ const MainPanel = () => {
 					</Layout>
 				</Scroller>
 			</Cell>
-			<Cell size="30%" className={css.previewSection}>
+			<Cell size="30%" className={css.previewSection} style={{backgroundColor: BGColor}}>
```
Comment (daniel-stoian-lgp) [inline sandstone/custom-skin/src/views/MainPanel.js:318]: this is not necesary.
you can simpy  have the change be applied directly from the css class

.previewSection {
		background-color: var(--sand-bg-color, #4c596a);
.......
....
}

File: sandstone/custom-skin/src/utils.js [hunk 2]
```diff
@@ -162,7 +162,7 @@ const generateColors = (background, text) => {
 	const bgColors = generateBGColors(background, 9);
 	const textColors = generateTextColors(text, 8);
 
-	return [textColors[0].toUpperCase(), background, text, bgColors[0].toUpperCase(), textColors[1].toUpperCase(), text,
+	return [textColors[0].toUpperCase(), bgColors[4].toUpperCase(), text, bgColors[0].toUpperCase(), textColors[1].toUpperCase(), text,
```
Comment (daniel-stoian-lgp) [inline sandstone/custom-skin/src/utils.js:165]: can you explain why is this change necesary?

File: sandstone/custom-skin/src/components/ColorPicker/ColorPicker.module.less [hunk 3]
```diff
@@ -1,3 +1,24 @@
...
+	height: 48px;
+	width: 576px;
+}
+
+.closeButton {
+	--sand-component-focus-text-color-rgb: 76, 80, 89;
+}
+
+.coloredButton {
+	border-radius: 25px;
```
Comment (daniel-stoian-lgp) [inline sandstone/custom-skin/src/components/ColorPicker/ColorPicker.module.less:17]: use multiple of 3px :)

**Review by daniel-stoian-lgp — COMMENTED** [review]
A few comments

File: sandstone/custom-skin/src/utils.js (hunk 2 above)
Comment (alexandrumorariu) [reply sandstone/custom-skin/src/utils.js:165]: The shadow was the same color as the background of our Live Preview making it imperceptible for auto.

**Review by daniel-stoian-lgp — APPROVED** [review]

**Review by adrian-cocoara-lgp — APPROVED** [review]

**Pull request merged**